src/main/java/eu/mihosoft/vrl/v3d/CSG.java

```diff
@@ -1100,13 +1337,16 @@ public MeshContainer toJavaFXMeshSimple() {
 
     /**
      * Returns the bounds of this csg.
+     * SIDE EFFECT bounds is created and simply returned if existing

```

Comment:
I made this change in order to make the bounds checks fast. Is there any reason i am not aware of that storing this value for the life of a CSG object is not valid?